oak_restricted_kernel/src/interrupts.rs

```diff
@@ -143,6 +143,8 @@ mutable_interrupt_handler_with_error_code!(
                 }
                 let leaf = stack_frame.rax as u32;
                 get_cpuid_for_vc_exception(leaf, stack_frame).expect("Error reading CPUID");
+                // CPUID instruction is 2 bytes long, so we advance the instruction pointer by 2.
+                stack_frame.rip = VirtAddr::new(stack_frame.rip.as_u64() + 2);
```

Comment:
I think you can just do `stack_frame.rip += 2;` here.

Reply:
I tried that originally but the compiler complained. Turns out it was just because it did not know what type "2" represented, so using `2u64` does work.